docs/introduction.rst

```diff
@@ -100,6 +92,11 @@ In addition, Fortran 2003 provides some object oriented programming facilities:
    * Procedure Polymorphism with Type-Bound Procedures
    * Enumerations compatible with C
 
+Technical Specification TS 29113, now part of Fortran 2019, introduced
```

Comment:
For this subsection, it might be helpful to briefly discuss how shroud deals with the problems/issues (when possible).

Reply:
I'm not exactly sure what you mean here.  I started a feature to indicate the standard your using so Shroud will generate different code.  However, it is still possible to generate code which may not compile with an older compiler.  Specifically, the assumed-type feature. It's been in a report since 2012 but only in the standard in 2018.  I tried to compile with the Portland compiler but it is stuck at 2003 and does not support assumed-type.

Reply:
The section is called "Issues" and discusses some things that wrappers/bindings have to contend with.

I think that section would benefit from some clarifications, since it is not clear to me how shroud deals with these problems, or what the reader is supposed to do with the information in that section.

It is entirely possible/likely that I am misunderstanding the section...

Reply:
Clarification: My comment was about the entire "Issues" section, rather than the specific sentence that I commented on.

Reply:
As you mentioned, the section is about things that wrappers/binders have to contend with. The implied message is that you don't want to do all that yourself when Shroud will solve the problem for you. I added a summary paragraph that maybe helps.
I also updated things a bit since the Fortran 2018 standard is official now.